Sources/SnowballSwiftKit/Views/SnowballButton.swift

```diff
@@ -8,11 +8,11 @@
 import SwiftUI
 
 public struct SnowballButton: View {
-    var title: String? = nil
-    var leftIcon: String? = nil
-    var leftSystemIcon: String? = nil
-    var rightIcon: String? = nil
-    var rightSystemIcon: String? = nil
+    var title: String?
+    var leftIcon: String?
+    var leftSystemIcon: String?
+    var rightIcon: String?
+    var rightSystemIcon: String?
```

Comment:
ditto what I said above, this is probably not the desired way to handle leading and trailing (oh yeah! never say left or right! because layouts with hstack get automatically flipped in arabic etc).